api/src/main/java/com/okta/idx/sdk/api/model/TokenType.java

```diff
@@ -19,7 +19,11 @@
 
     ACCESS_TOKEN("access_token"),
 
-    REFRESH_TOKEN("refresh_token");
+    REFRESH_TOKEN("refresh_token"),
+
+    ACTIVATION_TOKEN("activation_token"),
```

Comment:
This seems like it should be another enum type. It doesn’t really mesh with access token/refresh token.

Reply:
sure, will move it to a new enum type.

Reply:
done, updated PR.